Libplanet.Tests/Store/Trie/KeyValueStoreTest.cs

```diff
@@ -0,0 +1,104 @@
+using System;
+using System.Linq;
+using Libplanet.Store.Trie;
+using Xunit;
+
+namespace Libplanet.Tests.Store.Trie
+{
+    public abstract class KeyValueStoreTest
+    {
+        private const int PreStoredDataCount = 10;
+
+        private const int PreStoredDataKeySize = 16;
+
+        private const int PreStoredDataValueSize = 32;
+
+        protected IKeyValueStore KeyValueStore { get; set; }
+
+        protected Random Random { get; } = new Random();
+
+        private byte[][] PreStoredDataKeys { get; set; }
+
+        private byte[][] PreStoredDataValues { get; set; }
+
+        [Fact]
```

Comment:
`[Fact]`s should be `[SkippableFact]` as `RocksDB` is not available on some platforms.